Neptune.API/Controllers/OnlandVisualTrashAssessmentAreaController.cs

```diff
@@ -39,4 +39,33 @@ public ActionResult<OnlandVisualTrashAssessmentAreaDetailDto> GetByID([FromRoute
         var onlandVisualTrashAssessmentAreaDetailDto = OnlandVisualTrashAssessmentAreas.GetByID(DbContext, onlandVisualTrashAssessmentAreaID).AsDetailDto();
         return Ok(onlandVisualTrashAssessmentAreaDetailDto);
     }
+
+    [HttpPost]
+    [JurisdictionEditFeature]
+    [EntityNotFound(typeof(OnlandVisualTrashAssessmentArea), "onlandVisualTrashAssessmentAreaID")]
+    public ActionResult Update([FromBody] OnlandVisualTrashAssessmentAreaDetailDto onlandVisualTrashAssessmentAreaDto)
+    {
+        OnlandVisualTrashAssessmentAreas.Update(DbContext, onlandVisualTrashAssessmentAreaDto);
+        return Ok();
+    }
+
+    [HttpGet("{onlandVisualTrashAssessmentAreaID}/parcel-geometries")]
+    [JurisdictionEditFeature]
+    [EntityNotFound(typeof(OnlandVisualTrashAssessmentArea), "onlandVisualTrashAssessmentAreaID")]
+    public ActionResult<List<ParcelGeometrySimpleDto>> GetParcelGeometries([FromRoute] int onlandVisualTrashAssessmentAreaID)
+    {
+        var visualTrashAssessmentArea = OnlandVisualTrashAssessmentAreas.GetByID(DbContext, onlandVisualTrashAssessmentAreaID);

```

Comment:
can you rename visualTrashAssessment* to onlandVisualTrashAssessment* every where in your code?  just use ctrl r-r and let resharper do it.  I don't want these custom names